### ISSUE TITLE
The resulting model weights are different for each run

### ISSUE THREAD

**Issue by hatihoo** [AntixK/PyTorch-VAE#105]
Hi, I found the trained weight varies across different runs when running vanilla vae. If I use the same seed, the weight initialization is exactly the same, and the data loader and optimizer do the same thing, why are the resulting weights different?
